### ISSUE TITLE
Feature Request: Append Crop00001, etc. to original filename.

### ISSUE THREAD

**Issue by pr3sidentspence** [FrancoisMalan/DivideScannedImages#11]
I'm hoping to run tens of thousands of scans, front and back, through DSI. It could be difficult to match these front and backs if DSI doesn't scan them in a known order. It would be great if they could keep their original filename appended with a Cropped indicator.

**Comment by DarrellJonsson** [comment]
Good point. The order it grabs the objects still is a mystery, is it selecting and saving them by size or other characteristics. And yes if you are archiving thousands or in my case hundreds of separate images it makes the task difficult having the images output in what seems a random sequence.
